### DISCUSSION TITLE
Ethereum-provider CDN

### DISCUSSION THREAD

**Discussion by nicktan147** [WalletConnect/walletconnect-monorepo#2732] [Developer Support] [answered]
I see that most of the modules has cdn now, but not the ethereum-provider. Is there any plan to have a cdn for this package?

UPDATE: I mean the bundled package for ethereum provider

**Comment by pixhello** [comment]
I am looking for exactly the same thing, I hope Ethereum-provider CDN will be available soon.

**Comment by glitch-txs** [comment, marked as answer]
hey guys, could you try this?
https://github.com/glitch-txs/ethereum-provider-cdn

**Reply by nicktan147** [reply]
Works great! Thanks for the solution.
